_i18n/nl.yml

```diff
@@ -302,7 +302,8 @@ merchants:
 
   button7: Hosting
   button8: Juridische diensten
-  button9: Online Marketplaatsen
+  button9: Online Marktplaatsen
+

```

Comment:
stray new line

Reply:
Done, were actually two empty lines that needed deleting.